space-framework/pkg/space-manager/provider.go

```diff
@@ -94,15 +94,18 @@ func CreateProvider(c *controller, providerDesc *spacev1alpha1apis.SpaceProvider
 	defer c.lock.Unlock()
 
 	_, exists := c.providers[providerName]
+	c.logger.V(2).Info("createProvider", "name", providerDesc.Name)
 	if exists {
 		return nil, fmt.Errorf("provider %s already in the list", string(providerDesc.Spec.ProviderType))
 	}
 
+	c.logger.V(2).Info("createProvider", "name", providerDesc.Name)
 	configStr, err := getConfigFromSecret(*c.k8sClientset, providerDesc.Spec.SecretRef)
 	if err != nil {
 		return nil, err
 	}
 
+	c.logger.V(2).Info("createProvider", "name", providerDesc.Name)
```

Comment:
Why do we need this? we have identical print at the beginning and errors in case we fail in between